README.md

```diff
@@ -149,38 +149,47 @@ be used:
 
 ### Logging
 
-For clearer and machine-parseable log output, there in an extension to be able
-to add context to your logs which is output as
-[logfmt](https://brandur.org/logfmt) key/value pairs after the log message.
+For clearer and machine-parseable log output, the Rails logger is replaced by an
+extended logger to add context to your logs, which is output as
+[logfmt](https://brandur.org/logfmt) key/value pairs along with the log message.
+
+You can use the logger as usual:
 
 ```ruby
-# application.rb
+Rails.logger.info { 'hello world' }
+```
 
-require 'roo_on_rails/context_logging'
+From your console, the output will include the timestamp, severity, and message:
 
-class Application < Rails::Application
+```
+[2017-08-25 14:34:54.899]    INFO | hello world
+```
 
-  # add this block somewhere within the application class
-  logger = config.logger
-  if logger.nil?
-    logger = ActiveSupport::Logger.new($stdout)
-    logger.formatter = config.log_formatter
-  end
-  logger = ActiveSupport::TaggedLogging.new(logger) unless logger.respond_to?(:tagged)
-  config.logger = RooOnRails::ContextLogging.new(logger)
+In production (or whenever the output isn't a TTY), the timestamp is stripped
+(as it's provided by the logging pipes) and the output is fully valid `logfmt`:
 
-end
+```
+at=INFO msg="hello world"
 ```
 
-You can then add context using the `with` method:
+One can also add context using the `with` method:
 
 ```ruby
 logger.with(a: 1, b: 2) { logger.info 'Stuff' }
+# => at=INFO msg=Stuff a=1 b=2
```

Comment:
Not quite sure I understand the `at` name here? Wouldn't `level` be more obvious?

Reply:
I've been using `at=` for consistency with what Heroku does, and what [the original `logfmt` paper](https://brandur.org/logfmt) says.